cranelift/filetests/filetests/runtests/i128-icmp-overflow.clif

```diff
@@ -1,3 +1,4 @@
+test interpret
 test run
 target aarch64
```

Comment:
Does x86_64 support this too?

Reply:
And on the current `main` it fails in a different test:

```
Failed test: run: %icmp_of_i128(1, 170141183460469231731687303715884105727) == true, actual: false
```